activesupport/lib/active_support/core_ext/hash/except.rb

```diff
@@ -1,8 +1,14 @@
 class Hash
-  # Returns a hash that includes everything but the given keys.
-  #   hash = { a: true, b: false, c: nil}
-  #   hash.except(:c) # => { a: true, b: false}
-  #   hash # => { a: true, b: false, c: nil}
+  # Returns a hash that includes everything except given keys.
+  #   hash = { a: true, b: false, c: nil }
+  #   hash.except(:c) # => {a: true, b: false}
```

Comment:
I'd rather we follow our style guide. The hashes are the same, we're just choosing how to display them.